content/news/XXXX-XX-XX-mixxx-2-3-3-released.md

```diff
@@ -0,0 +1,30 @@
+title: Mixxx 2.3.3 released
+authors: Nikolaus Einhauser
+tags: 2.3, 2.3.3, release announcement
+comments: yes
+status: draft
+
+# Mixxx 2.3.3 Released
+
+Its that time again! 🎉  
```

Comment:
This emoji does not look very good IMHO, can we remove it?

Reply:
Only this particular instance or do you propose to remove all emojis?

Reply:
The others are OK, only this one looks like dirt on the screen :-) : 
![grafik](https://user-images.githubusercontent.com/1777442/173460209-44b3c98a-1a45-43c5-8476-97a376c8d310.png)

Reply:
Mhmmm, what do you think about using a :rocket: instead?

Reply:
Though I don't particularly like that either.

Reply:
This looks better here, but maybe we should just use nothing in this place. 
It is probably in issue of different imoji fonts and the small size. On my mobile it looks better.